Command/DebugCommand.php

```diff
@@ -117,30 +101,35 @@ protected function outputMailer(OutputInterface $output, $name)
             throw new \InvalidArgumentException(sprintf('The mailer "%s" does not exist.', $name));
         }
 
+        $tableHeaders = array('Property', 'Value');
+        $tableRows = array();
+
         $transport  = $mailer->getTransport();
         $spool = $this->getContainer()->getParameter(sprintf('swiftmailer.mailer.%s.spool.enabled', $name)) ? 'YES' : 'NO';
         $delivery = $this->getContainer()->getParameter(sprintf('swiftmailer.mailer.%s.delivery.enabled', $name)) ? 'YES' : 'NO';
         $singleAddress = $this->getContainer()->getParameter(sprintf('swiftmailer.mailer.%s.single_address', $name));
 
-        $output->writeln($this->getHelper('formatter')->formatSection('swiftmailer', sprintf('Mailer "%s"', $name)));
+        $output->title(sprintf('Configuration of the Mailer "%s"', $name));
```

Comment:
this triggers a fatal error if ``$output`` is null, which is allowed in your method signature.